src/internal/connector/onedrive/collection_test.go

```diff
@@ -59,6 +60,7 @@ func (suite *OneDriveCollectionSuite) TestOneDriveCollection() {
 	t := suite.T()
 	wg := sync.WaitGroup{}
 	collStatus := support.ConnectorOperationStatus{}
+	t1 := time.Now()
```

Comment:
nit: "now" is often the best name for any time.Now representation.
```suggestion
	now := time.Now()
```